app/app.go

```diff
@@ -828,6 +833,18 @@ func New(
 	app.ScopedIBCKeeper = scopedIBCKeeper
 	app.ScopedTransferKeeper = scopedTransferKeeper
 
+	// register streaming service
```

Comment:
There should be an `Info` log line here that lists all the keys available for streaming configuration. That way, people can more easily see what's available and know what keys to use when setting their config.

Reply:
Now that we're going to create our own `provenance.config` => #962, do we still want to do this here, somewhere else or not at all?

Reply:
This isn't really all that related to that issue. Regardless of where the config lives, people still need to know what strings are expected so that they can be properly configured.

Also, maybe this should panic if the configured key doesn't exist (rather than just logging an error and moving on).